test/dlc_tests/eks/tensorflow/inference/test_eks_tensorflow_inference.py

```diff
@@ -9,7 +9,7 @@
 import test.test_utils as test_utils
 
 
-@pytest.mark.model("mnist")
+@pytest.mark.model("simple")
```

Comment:
Is there a reason for straying away from a common test model of mnist?

Reply:
Oh ok.. This change is not needed. The model in the test is still mnist based. Will change the above back to mnist.